google-cloud-logging/src/main/java/com/google/cloud/logging/Logging.java

```diff
@@ -668,6 +668,197 @@ public static EntryListOption filter(String filter) {
    */
   ApiFuture<Boolean> deleteMetricAsync(String metric);
 
+  /**
+   * Creates a new exclusion in a specified parent resource. Only log entries belonging to that
+   * resource can be excluded. You can have up to 10 exclusions in a resource.
+   *
+   * <p>Example of creating the exclusion.
+   *
```

Comment:
nit: here and below:
Example of <text>. -> Example of <text>:

Reply:
I think `<p>Example of creating the exclusion.` better here, same thing found in other libraries as well.

Reply:
Yes, I know that in many places "." is used. But ":" fits better, IMHO. I got a comment from reviewer to change that. It's up to you to choose.

Reply:
done